dev/build/windows/makecoq_mingw.sh

```diff
@@ -2022,7 +2021,7 @@ fi
 
 list_files ocaml_coq
 
-make_addons
+#make_addons
 
 list_files_always ocaml_coq_addons
```

Comment:
```suggestion
 #list_files_always ocaml_coq_addons
```
I'll try this strategy blindly.